src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/ViewModels/ErrorViewModel.cs

```diff
@@ -0,0 +1,9 @@
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Api.ViewModels;
+
+public record ErrorViewModel : Common.ViewModels.ErrorViewModel
```

Comment:
There's some weaknesses with the common `ErrorViewModel` in that its fields are essentially all optional.

This is problematic as it means as this means that the view model is potentially going to be wrong in the generated docs.

There's also some other issues with the current generated docs in terms of required properties. Instead of doing this here, I'm going to try and tackle this in this subtask: https://dfedigital.atlassian.net/browse/EES-5057